.github/workflows/ci-tests.yml

```diff
@@ -5,6 +5,12 @@ env:
   PHP_VERSION: 8.1
   COMPOSER_VERSION: v2
 
+# Detect if this action is already running, and cancel it.
+# This most likely happened because a second push has been made to a branch.
+concurrency:
+  group: ${{ github.repository_id }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
I am not fond of canceling runs, because if a ci-test starts to fail we wont be able to trace exactly which commit hash that introduced the error.
It's not a biggie and I guess it is also about weighing cloud-resources/environmental awareness against the forensic possibility for pinpointing where an error occured.